test/unit-tests/mobile/devices-service.ts

```diff
@@ -469,6 +474,8 @@ describe("devicesService", () => {
 			assert.isFalse(devicesService.hasDevices, "Initially devicesService hasDevices must be false.");
 			iOSDeviceDiscovery.emit("deviceFound", iOSDevice);
 			androidDeviceDiscovery.emit("deviceFound", androidDevice);
+			let hostInfo = testInjector.resolve("hostInfo");
+			hostInfo.isDarwin = true;
```

Comment:
same here - the test should be platform independent - why should we set the OS to be macOS ?